compiler/damlc/tests/src/DamlcTest.hs

```diff
@@ -367,14 +367,14 @@ testsForDamlcTest damlc scriptDar _ = testGroup "damlc test" $
             assertBool
                 ("template creation coverage is reported correctly: " <> stdout)
                 (unlines
-                     [ "  never created: 1"
+                     [ "  internal templates never created: 1"
```

Comment:
I'm sorry I wasn't clear, in [my previous comment](https://github.com/digital-asset/daml/pull/15563#discussion_r1027909186) I was referring to keeping the output as `never created` while prepending the line `- Internal templates` to the expected infix. I now see that wouldn't quite work because the intermediate lines would become part of the test, however. For now I'd revert the last commit (otherwise the report gets too redundant imo). Later on, we probably want to bring back a `Report` type that's closer to the output and test _that_ instead (e.g. ``assertBool (...) (ContractIdentifier (...) "Foo:S" `elem` neverCreated (internalTemplates report)``)